docs/how-to-work-on-coding-challenges.md

```diff
@@ -304,6 +304,26 @@ function myFunc() {
 </details>
 ````
 
+## Testing Challenges
+
+Before you [create a pull request](how-to-open-a-pull-request.md) for your changes, you need to validate that the changes you have made do not inadvertently cause problems with the challenge.  You do this by performing the following steps:
```

Comment:
```suggestion
Before you [create a pull request](how-to-open-a-pull-request.md) for your changes, you need to validate that the changes you have made do not inadvertently cause problems with the challenge.  To test all challenges run `npm run test:curriculum`. To save time you can limit the tests to one challenge by performing the following steps:
```
That seems like a good way of getting across why you might want to run one test.  Reasonable?